pkg/manager/manager.go

```diff
@@ -278,6 +281,9 @@ func (m *Manager) Start() error {
 		return err
 	}
 
+	// TODO: placeholder for creation of the transactions store
+	var transactions transaction.Store
```

Comment:
Go ahead and create a store like Configuration store and start it. You don't need to put a place holder.

Reply:
I will rebase

Reply:
Done.